### ISSUE TITLE
Finding a chest, opening it.

### ISSUE THREAD

**Issue by kordostomas** [Darthfett/mineflayer-blockfinder#20]
Minecraft version 1.16.1
Hello, I found this example of chest opening and withdrawing items. https://github.com/PrismarineJS/mineflayer/blob/1abfe102766175b27fa39cb6cf587da6fc9a2d9a/examples/chest.js
But I couldn't find a chest.
So I found this plugin and tried the usage example.
I lowered the maxDistance, otherwise, it crashes. I placed a diamond ore next to the player, and it didn't find a single block. 
It just returns an empty array

If there's no way to fix this. Is there a way to open a chest at given coords? (constructing a Block)

**Comment by rom1504** [comment]
I advise you use mineflayer findBlocks which is now faster and better than
this package

**Comment by kordostomas** [comment]
@rom1504 I figured out a better way I think. Specifying the coords of the chest, finding a bock at the coords, opening it.
It will also open the right chest.

Anyway can you please refer to the documentation of findBlocks

This is the fix i've done 
```javascript
  for (var item of openChest.items()) {
    openChest.withdraw(item.type, null, item.count, (err) => {
      if (err) console.log(err)
    })
    await sleep(500) //otherwise it throws error: server rejected transaction
  }
```

**Comment by rom1504** [comment]
That's because you use a normal loop, you need to use the callback to
trigger the next iteration instead

**Comment by kordostomas** [comment]
@rom1504 Callback where? Should it be the callback on withdraw function? Can you provide an example?

**Comment by kordostomas** [comment]
Also are item types listed somewhere? Melon is 735. Doesn't seem like actual ids..

**Comment by rom1504** [comment]
Yes look mineflayer doc. And minecraft data one. Under PrismarineJS.
You're in the wrong project here
